config/application.yml

```diff
@@ -0,0 +1,6 @@
+HOST: "localhost:3000"
```

Comment:
oái, tạo 1 file clone .example cho file này em nhé
xóa git cache file này đi, toàn thông tin nhạy cảm mà đưa hết lên như này hở em
`git rm --cached config/application.yml`